src/custom/pages/Profile/index.tsx

```diff
@@ -206,6 +209,7 @@ export default function Profile() {
           </GridWrap>
           {!account && <Web3Status openOrdersPanel={() => console.log('TODO')} />}
         </GridWrap>
+        <ProgressBar onPercentageClick={() => setPercentage(111)} percentage={percentage} />
```

Comment:
Not quite there yet.

It should be like this:

```suggestion
        <ProgressBar onPercentageClick={setPercentage} percentage={percentage} />
```

Then now the state management is on this component, but triggered in the child via the callback